src/App.jsx

```diff
@@ -14,57 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+function GoodsList({ selectedGood, setSelectedGood }) {
+  return goods.map(item => (
+    <tr
+      data-cy="Good"
+      key={item}
+      className={item === selectedGood ? 'has-background-success-light' : ''}
+    >
+      <td>
+        {item === selectedGood ? (
+          <button
+            data-cy="RemoveButton"
+            type="button"
+            className="button is-info"
+            onClick={() => setSelectedGood('')}
+          >
+            -
+          </button>
+        ) : (
+          <button
+            data-cy="AddButton"
+            type="button"
+            className="button"
+            onClick={() => setSelectedGood(item)}
+          >
+            +
+          </button>
+        )}
+      </td>
+      <td data-cy="GoodTitle" className="is-vcentered">
+        {item}
+      </td>
+    </tr>
+  ));
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood('')}
```

Comment:
```suggestion
            onClick={reset}
```
Consider reusing this function